api/src/dto/ChangeReqAckDto.ts

```diff
@@ -1,16 +1,23 @@
-import { DocType, Uuid, AckStatus } from "../types";
-import { IsNotEmpty, IsString } from "class-validator";
+import { DocType, Uuid, AckStatus } from "../enums";
+import { Equals, IsEnum, IsNotEmpty, IsOptional, IsString } from "class-validator";
 
 /**
  * API acknowledgement to client after client submission of an ChangeReqDto document.
  */
 export class ChangeReqAckDto {
     @IsNotEmpty()
-    docId: Uuid; // Uuid of submitted ChangeReqDto to be acknowledged
+    @IsString()
+    reqId: Uuid; // Uuid of submitted ChangeReqDto to be acknowledged
```

Comment:
There is also a `@IsUUID` assertion which I think fits more for these cases.

Reply:
I left it as @IsString to allow using our current test data set. Might change it later though...

Reply:
I left it as @IsString to allow using our current test data set. Might change it later though...

Reply:
I left it as @IsString to allow using our current test data set. Might change it later though...